src/types/permissions.ts

```
@@ -0,0 +1,20 @@
+  "sme:admin",
+  "gateway:read",
+  "gateway:write",
+  "gateway:admin",
+  "enterprise:read",
+  "enterprise:write",
+  "enterprise:admin",
+]);
+
+export type PermissionScope = z.infer<typeof PermissionScopeEnum>;
+
+export const PermissionsArraySchema = z.array(PermissionScopeEnum);
```

ESLint

[error] Insert `⏎`

(prettier/prettier)